app/src/main/res/layout/dialog_sleep_timer.xml

```diff
@@ -28,13 +28,15 @@
             android:layout_height="wrap_content"
             android:layout_gravity="center"
             android:textAppearance="?android:textAppearanceLarge" />
+
     </FrameLayout>
 
     <CheckBox
-        android:id="@+id/checkBox"
+        android:id="@+id/should_finish_last_song"
         android:layout_width="wrap_content"
         android:layout_height="42dp"
-        android:layout_gravity="bottom|center_horizontal"
+        android:layout_gravity="bottom"
+        android:layout_margin="30dp"
```

Comment:
It looks like this now:
![screenshot](https://user-images.githubusercontent.com/12483662/50633965-0bd52e00-0f88-11e9-8998-68870e42daa2.png)
Is that ok?

Reply:
Can you try reducing the margin too?

Reply:
Is 20dp suitable?
I also shrinked the chinese text.
![default](https://user-images.githubusercontent.com/12483662/50694846-eaeb0680-1075-11e9-858c-feee622a4189.png)